psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -182,18 +187,19 @@ void DFHelper::initialize() {
     if (AO_core_) {
         prepare_AO_core();
         if (do_wK_) {
-            std::stringstream error;
-            error << "DFHelper: not equipped to do wK";
-            throw PSIEXCEPTION(error.str().c_str());
-            // TODO prepare_AO_wK_core();
+            prepare_AO_wK_core();
+        } else {
+            //			prepare_AO_core();
```

Comment:
else statment supposed to be empty?

Reply:
Perhaps this should go in possible added features. There's a formulation of J using density fitting where you only need 2 times `big_skips_[nbf_]` to do omega integrals in the case that `do_wK_` is true, but `do_K_` is false. You'd need to have a separate AO function in that case, so I was going to leave this bit in here to make the change easier to code up.